README.md

```diff
@@ -1,2 +1,92 @@
-# Hello-world
-A hello world website
+# HTML & CSS3 Course
```

Comment:
Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and follow best practices to write professional commit messages.

Reply:
Thank you for the review I capitalized the commit text on my last commit.